### ISSUE TITLE
Layout

### ISSUE THREAD

**Issue by Dmi3yy** [mnoskov/pagebuilder#5]
Для полного счастья еще не хватает layout тоесть сначала выбираем layout а уже потом выбираем блок какой нужно. 

p.s. Думаю стоит оставить это на позжее и собрать на это чутка денег с сообщества ибо очень крутая штука получается

**Comment by mnoskov** [comment]
Не понял, что в этом контексте значит layout. Что-то вроде группировки блоков?

**Comment by Dmi3yy** [comment]
Вот тут можно глянуть 
http://take.ms/4oXkP

https://www.modmore.com/contentblocks/live-demo/

**Comment by mnoskov** [comment]
Ну да, это в план.

**Comment by mnoskov** [comment]
Можно сделать что-то наподобие такого:
<img src="http://i92.fastpic.ru/big/2017/0616/e0/f883ec756d443510c99709c33ccad9e0.gif">

**Comment by Dmi3yy** [comment]
Круто

Отправлено с iPhone

**Comment by fourroses666** [comment]
Will this be possible? Custom options within a block.
I see an image here above where its possible to use columns.

I have a site where it will be very handy. 
It will reduce the ammount of blocks.

"A block select dropdown in a dropdown"

**Comment by mnoskov** [comment]
No, it has not been implemented yet, and don't know when it will be.
Maybe in next version :)

But it's not "columns in block", it is "blocks in columns"

**Comment by fourroses666** [comment]
Yea blocks in columns :) 

👍

**Comment by alekseiuva** [comment]
донатить куда?????

**Comment by vmoreira** [comment]
Hello, 

Is there any idea if something like this will be implemented?

Regards,
Vitor M.

**Comment by fourroses666** [comment]
Yup, really awesome if this will be possible

**Comment by fourroses666** [comment]
Till now I didn't really needed this but going to start with a project which really needs this. Hope you will need this too soon :-)

**Comment by fourroses666** [comment]
This looks really awesome;

![layout](https://user-images.githubusercontent.com/523389/51030258-1c456400-1599-11e9-88fd-14eb1d258328.gif)

**Comment by fourroses666** [comment]
Hi MNoskov, 

Did you try to implement something like it? 
Really like PageBuilder but also looking forward to be able to do these kind of things to make PageBuilder even better. Would reduce the amount of options allot.

> Можно сделать что-то наподобие такого:
> ![](https://camo.githubusercontent.com/5b26d63d2e7b5cf3e24d56a2c42cdbbd3af63d11/687474703a2f2f6939322e666173747069632e72752f6269672f323031372f303631362f65302f66383833656337353664343433353130633939373039633333636361643965302e676966)

**Comment by mnoskov** [comment]
Hi, Marc!
No, it was lost in the plans.
Don't know when I can get back to this because of the heavy workload.
Maybe it will be crowdfunded.

**Comment by fourroses666** [comment]
Oke got it, have a nice New years eve!

**Comment by Sybrenb** [comment]
This would be awesome

**Comment by fourroses666** [comment]
I really like the simplicity to configure PageBuilder but it does really needs a possibility to add multi-columns. Something like a pre-configured 2/3 column "simple" version like this: (`type: multicolumn`). Somekind of wrapper arround it. 

<img width="1423" alt="3-columns" src="https://user-images.githubusercontent.com/523389/80911141-c9b32c80-8d34-11ea-96bb-90bec8eb5f35.png">
